src/main/webapp/app/shared/util/core-submission/core-submission.spec.ts

```diff
@@ -0,0 +1,659 @@
+      ],
+      [
+        'mutations/0/mutation_effect/oncogenic',
+        { type: EvidenceEvidenceTypeEnum.Oncogenic, mutation, updateTime, gene, drugListRef, entrezGeneId },
+      ],
+      [
+        'mutations/0/mutation_effect/effect',
+        { type: EvidenceEvidenceTypeEnum.MutationEffect, mutation, updateTime, gene, drugListRef, entrezGeneId },
+      ],
+      [
+        'mutations/0/tumors/0/prognostic/description',
+        { type: EvidenceEvidenceTypeEnum.PrognosticImplication, tumor, mutation, updateTime, gene, drugListRef, entrezGeneId },
+      ],
+      [
+        'mutations/0/tumors/0/diagnostic/description',
+        { type: EvidenceEvidenceTypeEnum.DiagnosticImplication, tumor, mutation, updateTime, gene, drugListRef, entrezGeneId },
+      ],
+      [
+        'mutations/0/tumors/0/TIs/0/treatments/0',
+        {
+          type: EvidenceEvidenceTypeEnum.InvestigationalTherapeuticImplicationsDrugSensitivity,
+          tumor,
+          mutation,
+          updateTime,
+          gene,
+          ti: tiIs,
+          drugListRef,
+          treatment,
+          entrezGeneId,
+        },
+      ],
+      [
+        'mutations/0/tumors/0/TIs/1/treatments/0',
+        {
+          type: EvidenceEvidenceTypeEnum.InvestigationalTherapeuticImplicationsDrugResistance,
+          tumor,
+          mutation,
+          updateTime,
+          gene,
+          ti: tiIr,
+          drugListRef,
+          treatment,
+          entrezGeneId,
+        },
+      ],
+      [
+        'mutations/0/tumors/0/TIs/2/treatments/0',
+        {
+          type: EvidenceEvidenceTypeEnum.StandardTherapeuticImplicationsForDrugSensitivity,
+          tumor,
+          mutation,
+          updateTime,
+          gene,
+          ti: tiSs,
+          drugListRef,
+          treatment,
+          entrezGeneId,
+        },
+      ],
+      [
+        'mutations/0/tumors/0/TIs/3/treatments/0',
+        {
+          type: EvidenceEvidenceTypeEnum.StandardTherapeuticImplicationsForDrugResistance,
+          tumor,
+          mutation,
+          updateTime,
+          gene,
+          ti: tiSr,
+          drugListRef,
+          treatment,
+          entrezGeneId,
+        },
+      ],
+    ];
+    test.each(goodTests)('Path should parse as expected for path %s', (valuePath, args) => {
+      expect(pathToGetEvidenceArgs({ gene, valuePath, updateTime, drugListRef, entrezGeneId })).toEqual(args);
+    });
+    const badTests: [valuePath: string][] = [
+      ['bad/path'],
+      ['mutations/0/bad'],
+      ['mutations/300'],
+      ['mutations/bad/0/mutation_effect/'],
+      ['/'],
+      [''],
+    ];
+    test.each(badTests)('Should throw exception for path "%s"', (valuePath: string) => {
+      expect(() => pathToGetEvidenceArgs({ gene, valuePath, updateTime, drugListRef, entrezGeneId })).toThrow(Error);
+    });
+  });
+  describe('evidences mapping tests', () => {
```

Comment:
I think this group should also include the change content so in mapped evidence object, it has corresponding fielded updated. For instance, for `summary`, the description of the evidence needs to be updated. Otherwise it looks pretty similar to `Path parse tests`

Reply:
also need to include few missing cases mentioned in 'Path parse tests'

Reply:
I'm not quite sure what you mean by this. Can we have a call about it?

Reply:
check the `handleMutationEffect` function in type-specific-resolvers.ts